### PR TITLE
Add start and end time to Relationships

### PR DESCRIPTION
**Pull request by tsteenbe**
As was agreed in SPDX Technical Team Meeting of 2023-04-25.

### CODE REVIEW

**Review by zvr — CHANGES_REQUESTED** [review]
Agree, but all types should be `DateTime` (already in Core)

**Comment by tsteenbe** [comment]
@zvr  I fixed up the `xsd:dataTime` to `xsd:dateTime` in Relationship.md. Hope I get you right that `type: xsd:dateTime` should be `type: DateTime`?

I just copied what was used elsewhere, if everything need to be `type: DateTime` now I can fill a pull request to fix this across the entire specification.

**Comment by tsteenbe** [comment]
Slightly related, should the `created` property in Core not be renamed to `createdTime` so consistently across the spec all properties related to time will end in `Time`?

**Comment by tsteenbe** [comment]
Updated this PR to be aligned with @zvr  https://github.com/spdx/spdx-3-model/pull/278.

File: model/Core/Properties/endTime.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# startTime
```
Comment (maxhbr) [inline model/Core/Properties/endTime.md:3]:
```suggestion
# endTime
```

**Review by maxhbr — CHANGES_REQUESTED** [review]

File: model/Core/Classes/Relationship.md [hunk 2]
```diff
@@ -36,4 +36,11 @@ that one Element is related to one or more other Elements in some way.
   - type: RelationshipCompleteness
   - minCount: 0
   - maxCount: 1
-
+- startTime
+  - type: /Core/DateTime
```
Comment (zvr) [inline model/Core/Classes/Relationship.md:40]: No need for "/Core" here, this is in Core anyway.

**Review by zvr — CHANGES_REQUESTED** [review]

**Review by maxhbr — APPROVED** [review]
It does what it tries to do in a correct way. But I still think that not every relationship should have this feature. Would be good to fix that as a follow up.

I think that most generated documents describe some snapshot information and such time stamps add an orthogonal dimension that will create a lot of problems, inconsistencies and surprises.

**Review by kestewart — APPROVED** [review]
This may require further discussion but merging this for now, so we can consider wholistically.

**Pull request merged**